coss/settings.py

```diff
@@ -130,6 +130,10 @@
 COMPRESS_ENABLED = config('COMPRESS_ENABLED', default=True, cast=bool)
 COMPRESS_OFFLINE = config('COMPRESS_OFFLINE', default=True, cast=bool)
 COMPRESS_CACHE_BACKEND = config('COMPRESS_CACHE_BACKEND', default='default')
+COMPRESS_CSS_FILTERS = [
+    'compressor.filters.css_default.CssAbsoluteFilter',
+    'compressor.filters.cssmin.rCSSMinFilter'
```

Comment:
Same here.

Reply:
By default `COMPRESS_CSS_FILTERS` has only `css_default.CssAbsoluteFilter`. That's why I override it to add `rCSSMinFilter`. If you check css on stage right now you'll see it's not minified.